### ISSUE TITLE
attention mask problem

### ISSUE THREAD

**Issue by 3i-cao** [manycore-research/SpatialLM#69]
Hello,
I faced this warning in inference:
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:None for open-end generation.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

Does it affect to the results? any hint to fix it, thanks

**Comment by SamMaoYS** [comment]
Hi, it should not affect the reslut. A related [Qwen issue](https://github.com/QwenLM/Qwen3/issues/26#issuecomment-2358536071) for fixing.
